### PR TITLE
feat(wallet): support multiple outputs in create_vtt

### PR DESCRIPTION
**Pull request by mariocao**
This PR allows creating VTT with multiple outputs (address, value, time_lock).

### CODE REVIEW

**Comment by mariocao** [comment]
I fixed some tests and added a `map_err` to log if something fails. :smile:

File: wallet/src/actors/app/handlers/create_vtt.rs [hunk 1]
```diff
@@ -65,41 +67,51 @@ impl Handler<CreateVttRequest> for app::App {
...
-    Ok(pkh)
-}
+/// - destination address must be in the same network (testnet/mainnet)
+fn validate_output_addresses(
+    testnet: bool,
+    outputs: &[VttOutputParams],
+) -> Result<Vec<types::ValueTransferOutput>, app::ValidationErrors> {
+    outputs.iter().try_fold(vec![], |mut acc, output| {
+        types::PublicKeyHash::from_bech32(
+            if testnet {
```
Comment (girazoki) [inline wallet/src/actors/app/handlers/create_vtt.rs:96]: We can set the environment outside the loop right? that way we dont need to do the testnet mainnet comparison for every output

File: wallet/src/repository/wallet/tests/mod.rs [hunk 2]
```diff
@@ -1101,3 +1111,64 @@ fn test_create_vtt_with_locked_balance() {
         err,
     );
 }
+
+#[test]
+fn test_create_vtt_with_multiple_outputs() {
```
Comment (girazoki) [inline wallet/src/repository/wallet/tests/mod.rs:1116]: Is there a test to check for non-well serialized addresses?

Comment (mariocao) [reply wallet/src/repository/wallet/tests/mod.rs:1116]: I added tests in the `create_vtt` validation logic. :)

**Review by girazoki — APPROVED** [review]

**Pull request merged**